starwars/src/App.js

```diff
@@ -33,6 +35,7 @@ class App extends Component {
     return (
       <div className="App">
         <h1 className="Header">React Wars</h1>
+        <Card characters={this.state.starwarsChars} />
```

Comment:
`Card` might be the wrong term to use, since it seems like something that holds multiple cards